components/pedigree/resources/src/main/resources/pedigree/view/personHoverbox.js

```diff
@@ -158,6 +176,67 @@ define([
         },
 
         /**
+         * Generates alive, alive and well and deceased radio button section
+         *
+         * @method generateAliveWell
+         * @return {Raphael.el} Raphael element set
+         */
+        generateAliveWell: function() {
+            var node = this.getNode();
+            var lifeStatus = node.getLifeStatus();
+            var aliveandwellStatus = node.getAliveAndWell();
+            //generate 3 radio buttons
+            var labeles = ["Alive", "Alive & Well", "Deceased"];
+            var ticks = [];
+            var rects = [];
+            var height = this.getY()+this._height-this.getHoverBoxHeightExtension()-10;
+            var yTickIndex = 0;
+            var aliveAndWell = editor.getPaper().set();
+
+            for (var i = 0; i < 3; i++) {
+                var circle = this.generateCircle(this.getX()+15, height+20*i, false);
+                if ((lifeStatus == "alive" && !aliveandwellStatus && i == 0)
+                    || (aliveandwellStatus && i == 1)
+                    || (lifeStatus == "deceased" && i == 2) ) {
+                    yTickIndex = i;
+                }
+                var text = editor.getPaper().text(this.getX()+25, height+20*i, labeles[i]).attr(PedigreeEditorParameters.attributes.awLabel);
+                var rect = editor.getPaper().rect(this.getX()+5, height+20*i-10, this._width-20, 20, 1).attr(PedigreeEditorParameters.attributes.awRect);
+                rects[i] = rect;
+                var rowSet = editor.getPaper().set().push(circle, text, rect);
+                if (this._hidden && !this.isMenuToggled()) {
```

Comment:
it is better to do this check once outside the loop after the tick is created and do `aliveAndWell.hide()` which will hide everything